.github/workflows/remote-build-ios.yml

```diff
@@ -16,14 +16,12 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - project_root: ./
-            scheme: 'New Expensify Dev'
-            configuration: 'DebugDevelopment'
+            - scheme: 'New Expensify Dev'
+              configuration: 'DebugDevelopment'
```

Comment:
Forgive my ignorance but is the indentation correct here?

Reply:
Good catch! Indeed, indentation was wrong here 😅